utilities/test_suite/HOST/runTests.py

```diff
@@ -41,6 +41,51 @@
 caseMin = 0
 caseMax = 92
 
+augmentation_map = {
+    0: "brightness",
+    1: "gamma_correction",
+    2: "blend",
```

Comment:
This list is repeating for all 4 test suites in HOST and HIP with 8 maps.
Suggest placing just 4 maps called imageAugmentationMap, voxelAugmentationMap, audioAugmentationMap, miscAugmentationMap in the common.py file and access everywhere since its anyway included.

Modify the map's value to be a list of strings instead of a single string, so:
```
imageAugmentationMap = {
    0 : ["brightness", "HOST", "HIP"],
    ...
    49 : ["box_filter", "HIP"]
```

While calling print_case_list(imageAugmentationMap, backendType), also pass the backendType as a string parameter - "HIP" or "HOST", and while printing, check something like this:
if (backendType in <say the case 49 list name>):
    print(<case 49 list name [0]>)
The above common code would now print box_filter, for HIP, but not for HOST since its still in development.